src/Console/Commands/CrudModelBackpackCommand.php

```diff
@@ -91,31 +117,22 @@ public function handle()
                     // IDEs start counting from 1
 
                     // add CrudTrait
-                    array_splice($file_array, $position, 0, '    use \\'.$this->crudTrait.';');
+                    array_splice($lines, $position, 0, "    use \\{$this->crudTrait};");
 
                     // save the file
-                    $this->files->put($path, implode(PHP_EOL, $file_array));
+                    $this->files->put($path, implode(PHP_EOL, $lines));
 
                     // let the user know what we've done
-                    $this->info('Model already exists! We just added CrudTrait on it.');
+                    $this->error('Model already exists!');
+                    $this->info('We\'ve added CrudTrait on the Model.');
```

Comment:
```suggestion
                    $this->info('Model already existed. Added CrudTrait to it.');
```

I wouldn't output and error if the behaviour is what you'd expect...